test/generator-test.js

```diff
@@ -0,0 +1,179 @@
+var assert = require('yeoman-assert')
```

Comment:
I think we should probably open an issue to support CI in the repository to run these tests :)

Reply:
Do you want us to open this issue?

Reply:
https://github.com/theia-ide/generator-theia-extension/issues/47

Reply:
> #47

I've added it for you :)